docs/spelling_wordlist.txt

```diff
@@ -98,6 +98,7 @@ findable
 Fortran
 Frontend
 gapped
+gevent
```

Comment:
Out of curiosity, does it spellcheck if you use this?

```
``gevent``
```

(inline code in RST).

Reply:
this is the way